Stardrop/Utilities/External/NexusClient.cs

```diff
@@ -12,34 +12,34 @@
 using System.Text;
 using System.Text.Json;
 using System.Text.RegularExpressions;
+using System.Threading;
 using System.Threading.Tasks;
 
 namespace Stardrop.Utilities.External
 {
-    static class Nexus
+    public static class Nexus
     {
-        internal static int dailyRequestsRemaining;
-        internal static int dailyRequestsLimit;
+        private static readonly Uri _baseUrl = new Uri("https://api.nexusmods.com/v1/");
```

Comment:
One thing in particular I wondered about--the existing implementation has both an `HTTPS` and an `HTTP` endpoint, and only seemed to use the HTTPS one in a small handful of places. Are there some platform limitations on TLS handling here that I need to be aware of?

I defaulted the client here to be HTTPS-only (because it's 2024, and we should all be using HTTPS if we can), but please let me know if there are practical concerns here.

Reply:
I agree there is no reason to be using HTTP. For those old instances, I believe I may have copied them from their API example page but I do not recall.